app/crud.py

```
@@ -0,0 +1,63 @@
+        await db.refresh(todo)
+        return todo
+    except Exception as e:
+        await db.rollback()
+        raise HTTPException(status_code=500, detail=f"Error creating ToDo: {e}")
+
+
+async def get_todos(db: AsyncSession) -> list[ToDo]:
+    try:
+        result = await db.execute(select(ToDo))
+        return result.scalars().all()
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)